includes/functions/llms.functions.access.php

```diff
@@ -355,7 +355,7 @@ function llms_is_post_restricted_by_prerequisite( $post_id, $user_id = null ) {
 
 	// otherwise return false
 	// no prereq
-	return false;
+	return apply_filters( 'llms_is_post_restricted_by_prerequisite', false, $post_id, $user_id );
```

Comment:
new filter requires filter docblock

function needs `@since` tag added for new update

Reply:
For the @since, do you want 3.37.1 or the current version 3.37.0?

Reply:
Done

Reply:
you can use `@since [version]`
then `[version]` will be replaced by the release version when building the new release.